hoot-js/src/main/cpp/hoot/js/conflate/matching/ScriptMatchCreator.cpp

```diff
@@ -119,6 +124,12 @@ class ScriptMatchVisitor : public ConstElementVisitor
     {
       _getSearchRadius.Reset(current, Handle<Function>::Cast(value));
     }
+
+    // Point/Polygon is not meant to conflate any polygons that are conflatable by other conflation
```

Comment:
Throughout this class you'll see some custom behavior added to support `PointPolygon.js` with comments. The gist of it is that this class was written with the assumption that features with the same geometry types are being matched. For Point/Polygon that isn't the case.